### ISSUE TITLE
Post is not registered to both Authors

### ISSUE THREAD

**Issue by NetworkGuy149** [publishpress/PublishPress-Authors#441]
The plugin correctly changes the byline to multi-author and adds both Author boxes to the post. But it does not register the post to both Authors, only the first one.
![Multi-Author Post](https://user-images.githubusercontent.com/22349492/117809149-d1e95600-b22b-11eb-8a8e-9b0eed9c44f4.png)

When I use the WP Dashboard to see all posts published by author id, the post only shows up under the first Author, not the second. 
![Author 15 Hans Gillor](https://user-images.githubusercontent.com/22349492/117809822-a3b84600-b22c-11eb-8997-f6b52f944430.png)
![Author 23 Whynde Kuehn](https://user-images.githubusercontent.com/22349492/117809833-a74bcd00-b22c-11eb-94ac-dcb0362b8813.png)

## Expected Behavior

Expect multi-author posts to be credited to both authors.

## Current Behavior

Post is only registered with first Author in a multi-author post

## Context

We have Author pages for all of our contributors. Unless the post is registered to both Authors, only one gets credit on their author page.

**Comment by zakirsajib** [comment]
We have same issue in our React app. We used REST API to pull all posts of multi author. 
The post of The first author shows missing where he contributed. All other posts of the author shows up. 

Here is the example:
https://prairie-health-blog.vercel.app/author/admin/

It supposed to show 8 but it shows 7. 

Is there are any quick fix?

**Comment by andergmartins** [comment]
@zakirsajib yes, we still have limited support for the REST API. We are still not sure about how much we can fix but we have that on our plan, #509.

**Comment by andergmartins** [comment]
@NetworkGuy149 thanks for reporting the issue. We have a new release planned for this week. Please, could you test again and confirm you still have this issue?
